src/model/course.js

```diff
@@ -0,0 +1,19 @@
+class Course {
```

Comment:
Eu ainda fico com a impressão de que esta classe não é necessária. Poderia ser uma função:

``` javascript
// src/model/course.js

const course = (name, duration, description, author) => ({ 
  name, 
  duration, 
  description, 
  author 
})

module.exports = course

```

Reply:
Acredito que o time deve decidir se quer usar classes ou partir pra notação literal. O que for mais confortável para o time. Eu sou indiferente :joy:

Reply:
Bom ponto. Ficam aí as sugestões hehehe.

Reply:
o time decidiu pelas classes :dancer: